File: Basics_with_Python/PythonBasicCourse/1_First Steps In Coding/4_First_Steps_in_Coding_Exercises/more_exercises/10. Weather Forecast - Part 2.py
weather =  float(input())

if weather >= 26.00 and weather <= 35.00:
    print("Hot")
elif weather >= 20.1 and weather <= 25.9:
    print("Warm")
elif weather >= 15.00 and weather <= 20.00:
    print("Mild")
elif weather >= 12.00 and weather <= 14.9:
    print("Cool")
elif weather >= 5.00 and weather <= 11.9:
    print("Cold")
else:
    print("unknown")

File: Basics_with_JavaScript/34_First_Steps_in_Coding/More_Exersises/10._Weather_Forecast___Part_2.js
function vladi(input){;

    let temp = Number(input[0]);

    if (temp >= 26 && temp <= 35) 
        {
        console.log("Hot");
        }
    else if (temp >= 20.1 && temp <= 25.9) 
        {
        console.log("Warm");
        }
    else if (temp >= 15.00 && temp <= 20.00) 
        {
        console.log("Mild");
        }
    else if (temp >= 12.00 && temp <= 14.9) 
        {
        console.log("Cool");
        }
    else if (temp >= 5.00 && temp <= 11.9) 
        {
        console.log("Cold");
        }
    else
        {
        console.log("unknown");
        }

    };
    vladi([16.5]);
    vladi([8]);
    vladi([22.4]);
    vladi([26]);
    vladi([0]);
